CondTools/Ecal/python/copySimPulseShapeFromFile_cfg.py

```diff
@@ -0,0 +1,67 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TEST")
```

Comment:
i'm not sure about db practices.
In other areas/subsystems, files like this go to the /test directory

Reply:
I don't have a strong opinion, this file was initially created by Francesca Cavalari, who took care the creation of the DB format. I only did minor adjustments on the format itself, but not on the files and their structure throughout the various DB related folders